### DISCUSSION TITLE
Unity 3D with Clappr

### DISCUSSION THREAD

**Discussion by PanderaStudios** [clappr/clappr#2097] [Q&A]
Good afternoon, could you tell me if Clappr would run on the Unity 3D engine?

**Comment by pedrochamberlain** [comment]
Hello @PanderaStudios! Although I haven't tested your scenario, I believe it is possible to run Clappr on Unity's 3D engine.

As far as I'm aware, Unity does not have a native HTML renderer, but you can exhibit a webpage in the engine with plugins like [3D WebView](https://developer.vuplex.com/webview/overview) or [Unity WebView](https://github.com/gree/unity-webview).
